package.json

```diff
@@ -42,29 +42,14 @@
   "devDependencies": {
     "babel-cli": "6.24.1",
     "babel-eslint": "7.2.3",
-    "babel-plugin-check-es2015-constants": "6.22.0",
-    "babel-plugin-syntax-async-functions": "6.13.0",
     "babel-plugin-syntax-async-generators": "6.13.0",
-    "babel-plugin-transform-class-properties": "6.24.1",
```

Comment:
I think it's defined here - your change just caused the line to appear added since a `^` was added to the version